diff --git a/lib/prettier-plugin.cjs.js b/lib/prettier-plugin.cjs.js
index 96089b8baa0d4fc6defe1ac912bd70e708fbb72f..1b5e608eaa82935820d3ffc522296d48888dface 100644
--- a/lib/prettier-plugin.cjs.js
+++ b/lib/prettier-plugin.cjs.js
@@ -739,7 +739,7 @@ var TOKEN;
     });
     TOKEN.NON_PP_IDENTIFIER = chevrotain.createToken({
         name: "NON_PP_IDENTIFIER",
-        pattern: /[a-z]\w*/i,
+        pattern: /[a-z_]\w*/i,
         categories: TOKEN.IDENTIFIER,
     });
     TOKEN.PP = chevrotain.createToken({